css/main.css

```diff
@@ -194,8 +194,14 @@ li {
   }
 }
 
-.callout {
-  background: #f6f9fc;
+.callout-blue {
+  background: #ebf5ff;
+  padding: 20px;
```

Comment:
I know this was already there in the styles, but we can use `var(--spacing)` since it is set to 20px;